Doppler.ContactPolicies.Data.Access/Repositories/ContactPoliciesSettings/ContactPoliciesSettingsRepository.cs

```diff
@@ -36,16 +36,79 @@ public async Task<Entities.ContactPoliciesSettings> GetContactPoliciesSettingsAs
                 (await multipleAsync.ReadAsync<Entities.ContactPoliciesSettings>()).FirstOrDefault();
 
             if (contactPoliciesSettings == null)
+            {
                 return null;
+            }
+
             if (contactPoliciesSettings.IdUser == null)
+            {
                 return contactPoliciesSettings;
+            }
```

Comment:
Why are you changing the code formatting here and at the same time you are choosing a different format in `ContactPoliciesService.InsertContactPoliciesSettingsAsync`?

![image](https://user-images.githubusercontent.com/1157864/123634016-4a15d600-d7f0-11eb-88ed-8321e5004415.png)

Reply:
Not so important. I am closing this conversation...